data/presets/amenity/school.json

```diff
@@ -1,41 +1,34 @@
 {
-    "icon": "temaki-school",
-    "fields": [
-        "name",
-        "operator",
-        "operator/type",
-        "address",
-        "grades",
-        "religion",
-        "denomination",
-        "website"
-    ],
-    "moreFields": [
-        "capacity",
-        "charge_fee",
-        "email",
-        "fax",
-        "fee",
-        "gnis/feature_id-US",
-        "internet_access",
-        "internet_access/ssid",
-        "level",
-        "phone",
-        "polling_station",
-        "wheelchair"
-    ],
-    "geometry": [
-        "area",
-        "point"
-    ],
-    "terms": [
-        "academy",
-        "elementary school",
-        "middle school",
-        "high school"
-    ],
-    "tags": {
-        "amenity": "school"
-    },
-    "name": "School Grounds"
+  "icon": "temaki-school",
+  "fields": [
+    "name",
```

Comment:
Looks like your editor changed the formatting on these files. Running `npm run build` will change it back to make the diff cleaner.

Reply:
You’ll need to run the build anyways to update a couple files that are automatically derived from these JSON files.

Reply:
Thanks, just did but it only generated the .yaml file, not change the formatting. 

Are there editor setting recommendations somewhere? I just force-pushed a version that is a bit better and created https://github.com/openstreetmap/id-tagging-schema/issues/333 to discuss it further.

Reply:
Oh you’re right, I guess I’m just used to the automatic formatting that name-suggestion-index does and assumed it was set up the same way here.